### PR TITLE
[DOC-379] AWS Secret manager setup instructions

### PR DESCRIPTION
**Pull request by nadegepepin**

### CODE REVIEW

File: doc/docs/2.0.x/deploy-manage/deploy/aws-secret-manager.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+Update the file below with your `serviceAccountName` and  `secretProviderClass` before you run a `kubectl apply -f syncerpod.yaml`
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: secret-syncer
+spec:
+  containers:
+  - name: ubuntu
```
Comment (jrockway) [inline doc/docs/2.0.x/deploy-manage/deploy/aws-secret-manager.md:195]: This isn't ubuntu.

File: doc/docs/2.0.x/deploy-manage/deploy/aws-secret-manager.md [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+    image: busybox:latest
```
Comment (jrockway) [inline doc/docs/2.0.x/deploy-manage/deploy/aws-secret-manager.md:196]: We might consider `gcr.io/pause` here, a container designed only for sleeping.

Comment (nadegepepin) [reply doc/docs/2.0.x/deploy-manage/deploy/aws-secret-manager.md:196]: See #7597

File: doc/docs/2.0.x/deploy-manage/deploy/aws-secret-manager.md (hunk 1 above)
Comment (nadegepepin) [reply doc/docs/2.0.x/deploy-manage/deploy/aws-secret-manager.md:195]: See https://github.com/pachyderm/pachyderm/pull/7597

**Pull request merged**